mailpoet/assets/js/src/types/index.ts

```diff
@@ -1,9 +1,7 @@
-import { ComponentPropsWithoutRef } from 'react';
```

Comment:
@JanJakes Ha. Thanks for mentioning the `@types/wordpress__components` I see I haven't removed the package. I skipped removing it because it completely broke the premium build. Somehow, the imports from `@wordpress/components `that are exposed from the free plugin don't work if I remove the package. I had a lot of issues to fix and completely forgot about this one. I'll try to look into it. 

Also, I don't understand what you are reffering to in "Maybe we discussed this, but are now both needed?" What is the second thing?